cmd/brew-test-bot.rb

```diff
@@ -1466,6 +1466,7 @@ def test_ci_upload(tap)
       if ARGV.include?("--dry-run")
         puts "git push --force #{remote} master:master :refs/tags/#{git_tag}"
       else
+        safe_system "git", "reset", "--hard", "origin/master"
```

Comment:
Can you add this to `--dry-run` above too so it's obvious this command is run? Thanks!

Reply:
> it seems we are already on master.

Are we always on `master` at this point`, or only on Azure? I'm concerned that on other CI systems we're already on the pull request branch at this point. If Azure is on the pull request branch, but the pull request branch is named `master`, it needs to be renamed.